dist/s_win

```diff
@@ -67,7 +67,7 @@ win_filelist()
 	echo 'src/os_win/os_thread.c'
 	echo 'src/os_win/os_time.c'
 	echo 'src/os_win/os_vsnprintf.c'
-	echo 'src/os_win/os_yield.c') < filelist | sort > $t
+	echo 'src/os_win/os_yield.c') | sort > $t
```

Comment:
How do you feel about marking the POSIX-only files in `dist/filelist` with a condition of `POSIX` and including the Windows-specific files with a condition of `WINDOWS` so we don't need `build_win/filelist.win`?